charts/jaeger/README.md

```diff
@@ -160,8 +203,72 @@ data:
 ```
 
 ```bash
-kubectl apply -f jaeger-tls-cfgmap.yaml
-helm install jaeger jaegertracing/jaeger --values jaeger-values.yaml
+$ kubectl apply -f jaeger-tls-cfgmap.yaml
+$ helm install jaeger jaegertracing/jaeger --values jaeger-values.yaml
+```
+
+### Ingester and Kafka
+
+There are additional components which can also be installed with variety of choide for Kafka similar to Cassandra and Elasticsearch
+
+![Jaeger with Ingester](jaeger-with-ingester.png)
+
+**New Kafka cluster**
+
+Ingester enablement plays a key role in applying configurations in Collector to use Kafka as storage. You still need to provide additional option for Cassandra/Eleasticsearch which here are not written for brevity
+
+```bash
+$ helm install jaeger jaegertracing/jaeger \
+  --set provisionDataStore.kafka=true \
+  --set ingester.enabled=true
+```
+
+**Using an existing Kafka cluster**
+
+You can use an exisiting Kafak cluster with jaeger too
+
+```bash
+$ helm install jaeger jaegertracing/jaeger \
+  --set provisionDataStore.kafka=false \
+  --set ingester.enabled=true \
+  --set storage.kafka.broker={<BROKER1:PORT>,<BROKER2:PORT>} \
+  --set storage.kafka.topic=<TOPIC>
+```
+
+## Examples
+
+### Chart with New Cassandra, Ingester and New Kafka
+
+```bash
+$ helm install jaeger jaegertracing/jaeger \
+  --set provisionDataStore.kafka=true \
+  --set ingester.enabled=true
+```
+
+### Chart with Existing Cassandra, Ingester and Existing Kafka
+
+```bash
+$ helm install jaeger jaegertracing/jaeger \
+  --set ingester.enabled=true \
+  --set provisionDataStore.cassandra=false \
+  --set storage.cassandra.host=<HOST> \
+  --set storage.cassandra.port=<PORT> \
+  --set storage.cassandra.user=<USER> \
+  --set storage.cassandra.password=<PASSWORD> \
+  --set storage.kafka.broker={<BROKER1:PORT>,<BROKER2:PORT>} \
+  --set storage.kafka.topic=<TOPIC>
+```
+
+### Chart with Existing Cassandra, Ingester and New Kafka
+
+```bash
+$ helm install jaeger jaegertracing/jaeger \
```

Comment:
```suggestion
helm install jaeger jaegertracing/jaeger \
```

`$` is not needed if no ouput is shown as per https://github.com/DavidAnson/markdownlint/blob/v0.18.0/doc/Rules.md#md014